modules/libcom/src/osi/os/WIN32/osdFindSymbol.c

```diff
@@ -28,19 +47,46 @@ epicsShareFunc void * epicsLoadLibrary(const char *name)
 
 epicsShareFunc const char *epicsLoadError(void)
 {
-    static char buffer[100];
+    STORE char buffer[100];
+    DWORD n;
 
-    FormatMessage(
+    n = FormatMessage(
         FORMAT_MESSAGE_FROM_SYSTEM,
         NULL,
         epicsLoadErrorCode,
         0,
         buffer,
         sizeof(buffer)-1, NULL );
+
+    /* n - number of chars stored excluding nil.
+     *
+     * Some messages include a trailing newline, which we strip.
+     */
+    for(; n>=1 && (buffer[n-1]=='\n' || buffer[n-1]=='\r'); n--)
+        buffer[n-1] = '\0';
+
```

Comment:
If FormatMessage() fails n == 0 so a previous message written to buffer may be returned. Maybe something like    epicsSnprintf(buffer, sizeof(buffer), "epicsLoadError: Unable to format WIN32 error code %lu", epicsLoadErrorCode)   in that case